src/components/ArenaProofModal.js

```diff
@@ -70,72 +77,95 @@ export default function ArenaProofModal({
   };
 
   return (
-    <div>
-      <Modal show={show} onHide={handleClose}>
-        <Modal.Header closeButton>
-          <Modal.Title>{I18n.get("submitProof")}</Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          {view === "submit" ? (
-            <>
-              <FormGroup bsSize="large" controlId="trashTalk">
-                <ControlLabel>{I18n.get("trashTalkPSA")}</ControlLabel>
-                <FormControl
-                  type="text"
-                  onChange={handleChange}
-                  value={formData.trashTalk}
-                />
-              </FormGroup>
-              <h3>{I18n.get("attachment")}</h3>
-              <input type="file" onChange={(evt) => onChange(evt)} />
-              <br />
-              <div className="flex">
-                <Button onClick={handleClose}>{I18n.get("close")}</Button>
-                <LoaderButton
-                  onClick={() => {
-                    propsHandleChange(
-                      activeMission,
-                      activeIndex,
-                      day,
-                      formData.key,
-                      activeSprintId,
-                      `${user.fName} just completed ${activeMission.title}.${
-                        formData.trashTalk.length > 0
-                          ? `They also said: "${formData.trashTalk}"`
-                          : ""
-                      } `
-                    );
-                    setFormData({ trashTalk: "" });
-                    setFormData({ ...formData, key: "" });
-                    handleClose();
-                  }}
-                  size="large"
-                  text={I18n.get("submitProof")}
-                  loadingText={I18n.get("creating")}
-                  // ? Is there a reason this is commented?
-                  // disabled={!this.validateForm()}
-                  isLoading={loading}
-                />
-              </div>
-            </>
+    <Dialog
+      style={{
+        margin: "auto",
+      }}
+      fullWidth
+      open={show}
+      onClose={handleClose}
+      keepMounted
+      hideBackdrop={false}
+    >
+      <DialogTitle>
+        {I18n.get("submitProof")}
+        <IconButton
+          aria-label="close"
+          onClick={handleClose}
+          sx={{
+            position: "absolute",
+            right: 8,
+            top: 8,
+          }}
+        >
+          <MdClose />
+        </IconButton>
+      </DialogTitle>
+      {view === "submit" ? (
+        <>
+          <DialogContent>
+            <FormGroup bsSize="large" controlId="trashTalk">
```

Comment:
Would you please replace the `react-bootstrap` FormGroups and such with `react-hook-form` implementation, that uses MUI styling? For context, you can look at this pull-request that was submitted by @vmcodes to replace the forms in `Login.js` that should give you all of the information you need! https://github.com/mikhael28/paretOS/pull/136/files

Reply:
https://react-hook-form.com/

Reply:
Let me know if you have any trouble with this @AleksandrKurilov

Reply:
@mikhael28 Yeah, thank you, I acepted your invitation.

Also I've done move to react-hook-form, but I have a question:
The first about DialogLabel. The mui TextField's contains label text inside itself
Like this:
![5](https://user-images.githubusercontent.com/15705764/164075911-0ba60e73-8b2a-4774-9b02-c122e0cb708d.PNG)
The label FileInput below is out of input, that's why I made the same label for "trashTalk" field.
Now it looks like this:
![4](https://user-images.githubusercontent.com/15705764/164076698-6cb6afee-8404-43bb-9b13-9c65ad215999.PNG)

Also I deleted some fields from FormState (like athleteNotes, github and so) because they were unused.

Reply:
I like it, looks good! Also, great job removing unused form field state, we don't really need the `validate` function either. If you can, can you change SUBMIT PROOF to 'Submit Proof'?

Reply:
Yes, of course. I've done it. Also I moved out inline style into useStyle hook, I think so it be better.